### PR TITLE
Updating index.rst

### PR DESCRIPTION
**Pull request by Zuhayer**
Updating the references of app/console with bin/console.

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -64,8 +64,8 @@ Usage
 
 .. caution::
 
-    If your application is based on Symfony 3, replace ``php app/console`` by
-    ``php bin/console`` before executing any of the console commands included
+    If your application is based on Symfony <2.5.*, replace ``php bin/console`` by
```
Comment (xabbuh) [inline Resources/doc/index.rst]: should be "replace [...] **with**"

Comment (stof) [inline Resources/doc/index.rst]: why 2.5 ? Any 2.x version of Symfony uses `app/console` by default.

Comment (Zuhayer) [inline Resources/doc/index.rst]: @stof The directory structure of Symfony was changed from version 2.5 (google search), that is why I chose 2.5.*. 

I have only worked in 2.1, 3.0, 3.1, so I am not sure if 2.6-8 used app/console or bin/console.

Comment (xabbuh) [inline Resources/doc/index.rst]: @Zuhayer The directory structure of the Symfony Standard Edition was changed with Symfony 3.

**Pull request closed without merging**